test/test_data_handling.py

```diff
@@ -2,5 +2,230 @@
 This file contains all testing functions for the repository's data_handling
 module.
 
-Author(s): Kate Habich, Michael Plunkett
+Author(s): Chanteria Milner
 """
+
+import pandas as pd
+import json
+from math import isnan
+from data_handling.clean_ansirh import (
+    clean_ansirh,
+    set_default_types,
+)
+from data_handling.munge_ansirh import (
+    split_by_state,
+    translate_code_to_state,
+    split_by_zip,
+    make_row_dicts,
+)
+
+
+DATA = {
+    "facility ID": [1, 2],
+    "name": ["Clinic 1", "Clinic 2"],
+    "city": ["Chicago", "Dallas"],
+    "state": ["IL", "TX"],
+    "zip code": [60637.0, 72001.0],
+    "open in 2021": [float("nan"), "No"],
+    "provided abortions in 2021": [" YES", "no"],
+}
+
+ROWS = [
+    {
+        "facility ID": 1,
+        "name": "Clinic 1",
+        "city": "Chicago",
+        "state": "IL",
+        "zip code": 60637.0,
+        "open in 2021": float("nan"),
+        "provided abortions in 2021": " YES",
+    },
+    {
+        "facility ID": 2,
+        "name": "Clinic 2",
+        "city": "Dallas",
+        "state": "TX",
+        "zip code": 72001.0,
+        "open in 2021": "No",
+        "provided abortions in 2021": "no",
+    },
+]
+
+CORRECT_DEFAULT_TYPES = {
+    "facility ID": 0,
+    "name": None,
+    "city": None,
+    "state": None,
+    "zip code": 0.0,
+    "open in 2021": None,
+    "provided abortions in 2021": None,
+}
+
+DF = pd.DataFrame(DATA)
+
+STATES = [
```

Comment:
Don't we have files for these? It'd be better to use those than to list them all here, imo. We can put them as exceptions in the `.gitignore` file.